### DISCUSSION TITLE
Un-opped players can unclaim other people's claims

### DISCUSSION THREAD

**Discussion by kwoby** [GriefPrevention/GriefPrevention#1918] [Q&A] [answered]
Hi, any player in my minecraft server is able to unclaim other's claims just by standing in it and typing /unclaim. I've never had this issue before. I also don't think it's because of luckperms permissions because the only two permissions the players have are griefprevention.createclaims and griefprevention.deleteclaims. If anyone knows how to fix this please let me know.

**Comment by Jikoo** [comment, marked as answer]
https://docs.griefprevention.com/permissions/
> griefprevention.deleteclaims | Allows the player to delete and resize other player’s claims

`griefprevention.claims` is the general perm for pretty much all basic functionality.
